intro/m_exploratory_DA.Rmd

```diff
@@ -89,7 +89,9 @@ First we need to load our data, which we created in the module on [joins and bin
 ```{r import-jb-data}
 
 # groundwater level measurements joined to stations, perforations, and CES data
-gwl <- read_rds(here("data", "sacramento_gw_data_w_calenviro.rds"))
+
+# load imports an object named "gwl"
+load(here("data", "sacramento_gw_data_w_calenviro.rda"))
```

Comment:
I hardly ever use `load()`. As long as this line loads the named `sacramento_gw_data_w_calenviro.rda` file as an object named `gwl` we're in good shape. If this is the case, it may be worth mentioning to the class where the "gwl" name comes from and how the `.rda` object was created.

Reply:
Okay, sounds like a teachable moment, which is good. The only thing I would verify is:

- [ ] verify that `load(here("data", "sacramento_gw_data_w_calenviro.rda"))` loads an object named `gwl` instead of `sacramento_gw_data_w_calenviro`

Reply:
yup I double checked this and resaved a version that imports `gwl` and not something else.